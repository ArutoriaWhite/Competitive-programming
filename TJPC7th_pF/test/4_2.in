5272162 998
4138 0
9353 0
27467 1
32435 1
41520 1
46900 1
47106 1
58528 0
86160 1
91687 1
92194 0
100340 0
108439 0
108728 0
125911 1
127283 0
140324 0
158113 0
162759 0
166511 1
168141 0
175187 0
175279 0
177746 1
181250 1
181435 1
184642 1
194459 0
194780 1
201410 1
210502 0
211267 1
216877 0
218471 1
219154 0
220400 1
221265 1
224156 0
224467 0
226803 0
228132 0
228421 0
242077 1
250537 1
250830 0
259568 1
263802 0
265919 1
271970 0
273431 1
279421 1
288436 0
295464 0
296810 1
300085 0
301752 0
311511 0
312588 0
327793 0
340182 0
360406 0
362964 0
364777 1
371935 0
375197 1
381132 0
382015 1
387309 1
391212 1
393201 1
414195 0
416991 0
426311 1
426558 1
428651 0
430699 0
437441 1
442974 0
446532 0
447132 0
457798 0
459337 1
468495 1
472614 0
473276 0
490708 0
494886 0
501687 1
507227 1
512604 0
513072 1
513881 0
518162 1
524571 1
525573 1
532063 1
532641 0
532873 0
540046 0
541345 0
545236 0
545564 0
548420 1
549547 1
550595 1
553656 0
554216 0
562410 1
566275 1
566955 0
566998 1
567111 1
582607 1
586346 1
588951 0
594391 1
596182 0
599382 1
601391 0
602452 0
606120 1
612544 0
613560 0
618736 0
629314 0
631150 1
635957 1
649055 1
675582 1
676425 0
680611 0
683310 1
692708 1
693706 0
699940 1
700416 0
707110 1
707945 1
712756 1
734025 1
736115 0
736749 1
740846 1
743826 1
752903 1
796231 0
801098 0
804348 1
812823 0
820301 1
825125 0
833489 1
837525 1
838370 0
839879 1
844527 0
849583 1
850329 1
851251 1
862305 1
867763 1
868800 0
881129 1
884024 1
895999 1
899733 0
904994 1
907362 1
911548 1
919725 1
927522 0
937422 0
942346 1
948690 1
950305 1
963693 0
963981 0
965387 0
978747 1
979530 0
982005 0
993513 1
998512 1
1003239 1
1007633 1
1009605 0
1009685 1
1021099 1
1021270 1
1026524 1
1029185 1
1037589 0
1038277 1
1044497 0
1056566 1
1060826 0
1062219 0
1063630 0
1069599 1
1071929 0
1083586 0
1085841 1
1094166 1
1100059 1
1101084 1
1105869 1
1106227 0
1108424 1
1114664 0
1135861 0
1141463 1
1142924 1
1146216 1
1149019 0
1153710 0
1156928 1
1157050 1
1157214 1
1158212 1
1163711 1
1181369 1
1184811 0
1188717 1
1189503 0
1189576 1
1193866 1
1197361 1
1203574 1
1208659 0
1211636 1
1217824 0
1227223 0
1232369 0
1236475 0
1244436 0
1244846 1
1247884 0
1253023 1
1255749 0
1261152 0
1265934 0
1285098 0
1287107 1
1293115 0
1293496 1
1294154 1
1295991 0
1302946 0
1308263 1
1313055 0
1319517 0
1327459 0
1327621 0
1336529 0
1338632 0
1344524 0
1348580 1
1353644 1
1360090 1
1365862 1
1370475 1
1375965 0
1377051 0
1387378 0
1392015 1
1395607 0
1400446 1
1400865 1
1411293 0
1414275 1
1422959 1
1427792 1
1434699 1
1453628 1
1455825 1
1457728 0
1459324 1
1462565 1
1466866 1
1467624 0
1485000 1
1487170 0
1491120 1
1493361 0
1507134 0
1509408 0
1522805 0
1525613 1
1526629 1
1542737 1
1545837 0
1564909 1
1570697 1
1584696 1
1588569 0
1595375 1
1595863 0
1600252 1
1602375 1
1602439 1
1609083 0
1609339 1
1619632 1
1635927 1
1640290 0
1651697 0
1662749 1
1667098 0
1670180 1
1673379 0
1677499 1
1679289 0
1682120 0
1694925 0
1723159 0
1725515 0
1728214 1
1728626 0
1729028 1
1729115 1
1731269 1
1737685 1
1738256 0
1742650 0
1763799 0
1764858 1
1765881 0
1771858 1
1775676 0
1782121 1
1787250 1
1789118 1
1791539 0
1792817 1
1796480 1
1796516 0
1805464 1
1805605 0
1829284 1
1835403 1
1835531 0
1836375 0
1837569 0
1850151 0
1855618 0
1856584 1
1865270 1
1882766 1
1883315 1
1888253 1
1897831 0
1899792 1
1901438 0
1906482 1
1907910 0
1919244 0
1922265 0
1930365 0
1939684 1
1940514 1
1940616 0
1952750 1
1955207 0
1959870 0
1977432 0
1980136 1
1982477 1
1989990 0
1990507 1
1996824 1
1998492 1
1999470 1
1999514 1
2000067 1
2003028 0
2011452 1
2019918 1
2024853 1
2028776 0
2029076 0
2029434 1
2029918 1
2036518 1
2039941 1
2043526 0
2044815 1
2047805 1
2049657 0
2053416 0
2053717 0
2054602 1
2059176 0
2060796 1
2063108 1
2065131 0
2094284 1
2099271 0
2110949 0
2122318 0
2126656 0
2134723 0
2137841 1
2152091 0
2154833 0
2155288 1
2156474 0
2173572 0
2173867 0
2178164 0
2183888 1
2184950 0
2187411 1
2197280 1
2198018 0
2206481 0
2208542 1
2217637 0
2218571 1
2227540 0
2229520 1
2235545 1
2238377 1
2247237 1
2249758 1
2251319 1
2251379 0
2253204 1
2260218 1
2273393 1
2283635 0
2293332 1
2303807 1
2310622 1
2318389 0
2323458 1
2332051 0
2333957 0
2335663 0
2336059 0
2350131 1
2361765 0
2372188 1
2383655 1
2386924 1
2387565 0
2387624 0
2392411 1
2393020 0
2395908 0
2397080 1
2397573 0
2403568 0
2413627 1
2415197 0
2417983 1
2422868 0
2435665 0
2439354 0
2443591 1
2445006 0
2446183 1
2449692 1
2450093 1
2465132 1
2469257 1
2470745 1
2482017 1
2482749 0
2498712 0
2502337 1
2505521 1
2506623 1
2518796 0
2536796 0
2538973 1
2540586 0
2544867 1
2547692 0
2554955 0
2557461 1
2557878 1
2583827 0
2584218 1
2587663 0
2589059 0
2597722 1
2612358 1
2613857 1
2615029 0
2615470 1
2615803 0
2620544 1
2632427 1
2642327 1
2651212 0
2655904 0
2657605 1
2668010 1
2671029 1
2672534 0
2680268 0
2681776 0
2684508 0
2689693 0
2691331 0
2692799 0
2696655 0
2700927 1
2712129 1
2712559 0
2716422 0
2723594 1
2730963 1
2734399 1
2735624 0
2738840 0
2751978 1
2752031 0
2753012 1
2765167 0
2773295 0
2782770 0
2787700 1
2788795 0
2804161 1
2804956 1
2810771 0
2818177 0
2824013 0
2834981 0
2837615 0
2843734 0
2844121 1
2844353 1
2846891 1
2859440 1
2859481 0
2861559 0
2864061 1
2868546 1
2871870 0
2872490 0
2874178 0
2877498 1
2883228 0
2887657 1
2888050 1
2894257 0
2894601 0
2896754 0
2898613 0
2901774 1
2905624 0
2920501 0
2928305 0
2939128 0
2945665 1
2947891 0
2953184 1
2962304 1
2965539 0
2967156 0
2970706 0
2970941 0
2972057 0
2975861 1
2983830 1
2990947 0
2993522 0
2994426 0
2995513 0
2996130 0
3006909 0
3010134 1
3013926 0
3017766 1
3031414 0
3041895 0
3046600 1
3048434 0
3062039 0
3062554 1
3063010 0
3070834 1
3077646 1
3080383 0
3085264 1
3100267 0
3116535 1
3123259 1
3133179 1
3139015 1
3140045 1
3140758 0
3142041 1
3151067 0
3153180 1
3165031 0
3177065 0
3183896 1
3189098 1
3192163 1
3204639 1
3204748 1
3208673 1
3243584 0
3248827 0
3253253 1
3255998 1
3256227 0
3261974 0
3262658 0
3273435 1
3301730 0
3302837 1
3308056 1
3312935 0
3314949 0
3316595 0
3316631 1
3322892 1
3333164 1
3333254 0
3333296 0
3343924 1
3345309 1
3357053 1
3359985 0
3369489 0
3371499 0
3375125 1
3379237 0
3379426 1
3389446 1
3393751 1
3399423 1
3402058 1
3407498 1
3408222 0
3412302 0
3418764 0
3419720 0
3420857 1
3424002 0
3425802 1
3426652 1
3428429 0
3433733 1
3441700 0
3448554 0
3452819 0
3467544 0
3468048 0
3469858 0
3474155 0
3475476 0
3481491 1
3485564 1
3489399 0
3491050 0
3502192 1
3502332 0
3530106 0
3532690 0
3537061 0
3541688 1
3541795 1
3546044 1
3552038 1
3557981 1
3564971 1
3569821 1
3570661 1
3576780 0
3581112 1
3588028 1
3589350 0
3598427 1
3613898 0
3618870 0
3635907 1
3658189 1
3665962 0
3677449 0
3679392 1
3684145 0
3689109 0
3705499 0
3708638 0
3712494 1
3712918 1
3713152 0
3713672 0
3714194 1
3725555 1
3727847 1
3729865 0
3744782 1
3750531 0
3757442 0
3768388 0
3773511 1
3776333 0
3782512 1
3784570 1
3786510 0
3787142 0
3790245 1
3802504 0
3803440 0
3810145 0
3812640 0
3820017 0
3823900 0
3824965 1
3830213 0
3833273 1
3840600 1
3841800 1
3843967 1
3846628 0
3848467 1
3850361 0
3853066 0
3859611 0
3863285 0
3865946 1
3869233 0
3878480 0
3882799 0
3883753 1
3884551 0
3886850 0
3892430 1
3893718 1
3899292 0
3904326 0
3909803 0
3912765 1
3926236 0
3929565 0
3935803 1
3941307 1
3941692 1
3943597 0
3950291 1
3951818 1
3961497 1
3964665 0
3974139 1
3974349 0
3975627 1
3980794 0
3980995 1
3983218 1
3984775 0
3991756 0
3993599 1
3998588 1
4017819 1
4022152 1
4033656 0
4035157 1
4041048 0
4052891 0
4055917 0
4056921 0
4062028 1
4068673 0
4068680 0
4072300 0
4074388 0
4076578 1
4077243 1
4091464 0
4097221 1
4105324 1
4112484 1
4136349 1
4141946 1
4144896 0
4147625 0
4150162 0
4150380 0
4151027 0
4152198 0
4156520 0
4157644 1
4165923 0
4169107 0
4176303 1
4176372 0
4180245 1
4186530 1
4194612 1
4199066 0
4201798 1
4206305 0
4207892 1
4215009 1
4216356 0
4217568 1
4222474 0
4227134 0
4231527 1
4232170 1
4236582 0
4238194 1
4238659 0
4251755 0
4254064 0
4255286 1
4261858 1
4271599 1
4271738 0
4273169 1
4274967 1
4282093 0
4293037 0
4294702 0
4301178 0
4306615 1
4322011 1
4326342 1
4331976 1
4332072 1
4337315 0
4339255 1
4350363 0
4353473 0
4356038 0
4356159 1
4364992 1
4368094 0
4372573 1
4379285 0
4380076 0
4381094 0
4382665 1
4383933 0
4388870 0
4397834 0
4402116 0
4403072 0
4408346 0
4409332 0
4409770 0
4420300 1
4422446 0
4422596 1
4429879 1
4434133 1
4447780 1
4449894 1
4473846 1
4499866 0
4513382 1
4518060 1
4518315 1
4518928 0
4519565 1
4527521 1
4530933 1
4533917 1
4543117 1
4554110 1
4560408 0
4570896 1
4573802 0
4577736 0
4578806 1
4579013 0
4580120 1
4592573 0
4594611 0
4595121 1
4599788 0
4601758 0
4606539 0
4607128 0
4611081 0
4612453 1
4623649 0
4627297 1
4630081 0
4632256 1
4634903 0
4650456 0
4651745 1
4655840 0
4657575 1
4660158 1
4675036 0
4680436 0
4698291 0
4713055 1
4714740 1
4717870 0
4718352 0
4724129 1
4726323 1
4730132 1
4731900 1
4734440 0
4742469 1
4742616 0
4756375 1
4761582 1
4767489 0
4769803 1
4772202 1
4773323 1
4774278 0
4774890 0
4778854 0
4778993 1
4786534 1
4787428 1
4795730 1
4810233 1
4831461 1
4841618 1
4869529 1
4870875 1
4877759 1
4878170 0
4879072 1
4888855 1
4908086 0
4913124 0
4916584 1
4927422 0
4934060 0
4935732 0
4940080 0
4948268 1
4949738 1
4955985 0
4958505 0
4958904 0
4964581 0
4967007 1
4969300 0
4971588 0
4978325 1
4980260 1
4986597 0
4998706 0
5000604 1
5002867 1
5002906 0
5004832 0
5008417 0
5013574 1
5013851 1
5036125 0
5036900 0
5041502 0
5055537 0
5056078 1
5061841 0
5066774 0
5068332 0
5069271 0
5069370 1
5080379 1
5088881 1
5089588 0
5098950 1
5104987 1
5106322 0
5112487 0
5114304 1
5125788 1
5126222 0
5129598 1
5149502 0
5166557 0
5186131 1
5193291 0
5199258 1
5200176 1
5202112 0
5211067 1
5221634 0
5223572 0
5224108 0
5224313 0
5227065 0
5236809 1
5243614 0
5244839 1
5245037 0
5248732 1
5254705 0
5255916 0
5257115 1
5262676 1
