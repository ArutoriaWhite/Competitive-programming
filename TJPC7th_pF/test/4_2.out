3541688 5237070
